Production/crab/crab_test_tupleProd.yaml

```diff
@@ -0,0 +1,11 @@
+config:
+  params:
+    cmsRunCfg: Production.py
+    maxEvents: 10
+    copyInputsToLocal: false
+    cmsRunOptions: "sampleType=MC,era=Run2_2018"
```

Comment:
you need to update params in all configs:
- add `cmsRunOptions `
- remove sampleType, era, storeFailed